README.md

```diff
@@ -1,15 +1,12 @@
 # Hello-Microverse
 
-
-# Project Name
--Microverse Hello
-> One paragraph statement about the project.
+> This project is about setting up linters and creating a hello-microverse page.
 
 ![screenshot](./photos/screenshot.png)
```

Comment:
- [ ] Please check this photo because it's not working.
I think the problem is the name of the image in the letter `s`.

Reply:
thanks fixed it 😄

Reply:
Ok, @Kayonga99 Kindly ask for another review with this Pull Request link don't create another Pull Request.
Have a great day 😊.